{"failedCount":1,"prefixedPath":"src\/test\/java\/modules\/get\/modules-get.feature","packageQualifiedName":"src.test.java.modules.get.modules-get","loopIndex":-1,"relativePath":"src\/test\/java\/modules\/get\/modules-get.feature","scenarioResults":[{"sectionIndex":0,"stepResults":[{"result":{"nanos":3439190500,"millis":3439.1905,"status":"passed"},"step":{"background":true,"line":4,"prefix":"*","index":0,"text":"call read(\"..\/..\/user\/token\/users-token-post.feature@token\")"},"callResults":[{"failedCount":0,"prefixedPath":"src\/test\/java\/modules\/get\/..\/..\/user\/token\/users-token-post.feature","packageQualifiedName":"src.test.java.modules.get.user.token.users-token-post","loopIndex":-1,"relativePath":"src\/test\/java\/modules\/get\/..\/..\/user\/token\/users-token-post.feature","scenarioResults":[{"sectionIndex":0,"stepResults":[{"result":{"nanos":2624000,"millis":2.624,"status":"passed"},"step":{"line":6,"prefix":"Given","index":0,"text":"url \"https:\/\/funcoeback-dev-ceqh.3.us-1.fl0.io\/users\/token\/\""}},{"result":{"nanos":45643000,"millis":45.643,"status":"passed"},"step":{"line":7,"prefix":"And","index":1,"text":"request {\"user\": \"ruben1\", \"password\": \"ruben123\"}"}},{"result":{"nanos":3208406900,"millis":3208.4069,"status":"passed"},"step":{"line":8,"prefix":"When","index":2,"text":"method post"},"stepLog":"22:34:08.356 request:\n1 > POST https:\/\/funcoeback-dev-ceqh.3.us-1.fl0.io\/users\/token\/\n1 > Content-Type: application\/json; charset=UTF-8\n1 > Content-Length: 39\n1 > Host: funcoeback-dev-ceqh.3.us-1.fl0.io\n1 > Connection: Keep-Alive\n1 > User-Agent: Apache-HttpClient\/4.5.13 (Java\/21.0.2)\n1 > Accept-Encoding: gzip,deflate\n{\"user\":\"ruben1\",\"password\":\"ruben123\"}\n\n22:34:11.273 response time in milliseconds: 2915\n1 < 200\n1 < date: Tue, 05 Mar 2024 03:34:09 GMT\n1 < server: envoy\n1 < content-length: 164\n1 < content-type: application\/json\n1 < x-envoy-upstream-service-time: 2299\n{\"access_token\":\"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoicnViZW4xIiwiZXhwaXJlcyI6MTcwOTcwOTY1MS41NzE4NzZ9.46Wx_K_eqfgT8ceQ0vgjDcZwnBmp0GFPBkO5DzEz9XQ\"}\n\n"},{"result":{"nanos":3120100,"millis":3.1201,"status":"passed"},"step":{"line":9,"prefix":"Then","index":3,"text":"status 200"}},{"result":{"nanos":19383300,"millis":19.3833,"status":"passed"},"step":{"line":10,"prefix":"*","index":4,"text":"def token = $.access_token"}}],"line":5,"description":"","durationMillis":3279.1773,"failed":false,"tags":["token"],"executorName":"main","name":"Post a token","startTime":1709609648016,"refId":"[1:5]","endTime":1709609651298,"exampleIndex":-1}],"callDepth":1,"name":"JWT Token Feature","description":"","resultDate":"2024-03-04 10:34:12 p.\u202Fm.","durationMillis":3279.1773,"passedCount":1}]},{"result":{"nanos":9081300,"millis":9.0813,"status":"passed"},"step":{"background":true,"line":5,"prefix":"*","index":1,"text":"header Authorization = \"Bearer \" + token"}},{"result":{"nanos":40791100,"millis":40.7911,"status":"passed"},"step":{"background":true,"line":6,"prefix":"*","index":2,"text":"print token"},"stepLog":"22:34:11.351 [print] eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoicnViZW4xIiwiZXhwaXJlcyI6MTcwOTcwOTY1MS41NzE4NzZ9.46Wx_K_eqfgT8ceQ0vgjDcZwnBmp0GFPBkO5DzEz9XQ \n"},{"result":{"nanos":457700,"millis":0.4577,"status":"passed"},"step":{"line":11,"prefix":"Given","index":0,"text":"url 'https:\/\/funcoeback-dev-ceqh.3.us-1.fl0.io\/modules\/all\/'"}},{"result":{"nanos":850855400,"millis":850.8554,"status":"passed"},"step":{"line":12,"prefix":"When","index":1,"text":"method get"},"stepLog":"22:34:11.353 request:\n2 > GET https:\/\/funcoeback-dev-ceqh.3.us-1.fl0.io\/modules\/all\/\n2 > Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoicnViZW4xIiwiZXhwaXJlcyI6MTcwOTcwOTY1MS41NzE4NzZ9.46Wx_K_eqfgT8ceQ0vgjDcZwnBmp0GFPBkO5DzEz9XQ\n2 > Host: funcoeback-dev-ceqh.3.us-1.fl0.io\n2 > Connection: Keep-Alive\n2 > User-Agent: Apache-HttpClient\/4.5.13 (Java\/21.0.2)\n2 > Accept-Encoding: gzip,deflate\n\n\n22:34:11.640 request:\n3 > GET https:\/\/funcoeback-dev-ceqh.3.us-1.fl0.io\/modules\/all\/\n3 > Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoicnViZW4xIiwiZXhwaXJlcyI6MTcwOTcwOTY1MS41NzE4NzZ9.46Wx_K_eqfgT8ceQ0vgjDcZwnBmp0GFPBkO5DzEz9XQ\n3 > Host: funcoeback-dev-ceqh.3.us-1.fl0.io\n3 > Connection: Keep-Alive\n3 > User-Agent: Apache-HttpClient\/4.5.13 (Java\/21.0.2)\n3 > Accept-Encoding: gzip,deflate\n\n\n22:34:11.845 request:\n4 > GET https:\/\/funcoeback-dev-ceqh.3.us-1.fl0.io\/modules\/all\/\n4 > Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoicnViZW4xIiwiZXhwaXJlcyI6MTcwOTcwOTY1MS41NzE4NzZ9.46Wx_K_eqfgT8ceQ0vgjDcZwnBmp0GFPBkO5DzEz9XQ\n4 > Host: funcoeback-dev-ceqh.3.us-1.fl0.io\n4 > Connection: Keep-Alive\n4 > User-Agent: Apache-HttpClient\/4.5.13 (Java\/21.0.2)\n4 > Accept-Encoding: gzip,deflate\n\n\n22:34:12.198 response time in milliseconds: 353\n4 < 500\n4 < date: Tue, 05 Mar 2024 03:34:12 GMT\n4 < server: envoy\n4 < content-length: 2600\n4 < content-type: text\/plain; charset=utf-8\n4 < x-envoy-upstream-service-time: 246\nTraceback (most recent call last):\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/middleware\/errors.py\", line 164, in __call__\n    await self.app(scope, receive, _send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/middleware\/exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 64, in wrapped_app\n    raise exc\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 53, in wrapped_app\n    await app(scope, receive, sender)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 762, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 782, in app\n    await route.handle(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 297, in handle\n    await self.app(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 77, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 64, in wrapped_app\n    raise exc\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 53, in wrapped_app\n    await app(scope, receive, sender)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 72, in app\n    response = await func(request)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/fastapi\/routing.py\", line 315, in app\n    content = await serialize_response(\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/fastapi\/routing.py\", line 155, in serialize_response\n    raise ResponseValidationError(\nfastapi.exceptions.ResponseValidationError: 1 validation errors:\n  {'type': 'model_attributes_type', 'loc': ('response',), 'msg': 'Input should be a valid dictionary or object to extract fields from', 'input': None, 'url': 'https:\/\/errors.pydantic.dev\/2.5\/v\/model_attributes_type'}\n\n\n\n"},{"result":{"nanos":141400,"errorMessage":"status code was: 500, expected: 200, response time in milliseconds: 353, url: https:\/\/funcoeback-dev-ceqh.3.us-1.fl0.io\/modules\/all\/, response: \nTraceback (most recent call last):\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/middleware\/errors.py\", line 164, in __call__\n    await self.app(scope, receive, _send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/middleware\/exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 64, in wrapped_app\n    raise exc\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 53, in wrapped_app\n    await app(scope, receive, sender)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 762, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 782, in app\n    await route.handle(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 297, in handle\n    await self.app(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 77, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 64, in wrapped_app\n    raise exc\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 53, in wrapped_app\n    await app(scope, receive, sender)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 72, in app\n    response = await func(request)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/fastapi\/routing.py\", line 315, in app\n    content = await serialize_response(\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/fastapi\/routing.py\", line 155, in serialize_response\n    raise ResponseValidationError(\nfastapi.exceptions.ResponseValidationError: 1 validation errors:\n  {'type': 'model_attributes_type', 'loc': ('response',), 'msg': 'Input should be a valid dictionary or object to extract fields from', 'input': None, 'url': 'https:\/\/errors.pydantic.dev\/2.5\/v\/model_attributes_type'}\n\n\nsrc\/test\/java\/modules\/get\/modules-get.feature:13","millis":0.1414,"status":"failed"},"step":{"line":13,"prefix":"Then","index":2,"text":"status 200"},"stepLog":"22:34:12.203 src\/test\/java\/modules\/get\/modules-get.feature:13\nThen status 200\nstatus code was: 500, expected: 200, response time in milliseconds: 353, url: https:\/\/funcoeback-dev-ceqh.3.us-1.fl0.io\/modules\/all\/, response: \nTraceback (most recent call last):\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/middleware\/errors.py\", line 164, in __call__\n    await self.app(scope, receive, _send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/middleware\/exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 64, in wrapped_app\n    raise exc\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 53, in wrapped_app\n    await app(scope, receive, sender)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 762, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 782, in app\n    await route.handle(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 297, in handle\n    await self.app(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 77, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 64, in wrapped_app\n    raise exc\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 53, in wrapped_app\n    await app(scope, receive, sender)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 72, in app\n    response = await func(request)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/fastapi\/routing.py\", line 315, in app\n    content = await serialize_response(\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/fastapi\/routing.py\", line 155, in serialize_response\n    raise ResponseValidationError(\nfastapi.exceptions.ResponseValidationError: 1 validation errors:\n  {'type': 'model_attributes_type', 'loc': ('response',), 'msg': 'Input should be a valid dictionary or object to extract fields from', 'input': None, 'url': 'https:\/\/errors.pydantic.dev\/2.5\/v\/model_attributes_type'}\n\n\nsrc\/test\/java\/modules\/get\/modules-get.feature:13\n"}],"line":10,"description":"","durationMillis":4340.5174,"failed":true,"error":"status code was: 500, expected: 200, response time in milliseconds: 353, url: https:\/\/funcoeback-dev-ceqh.3.us-1.fl0.io\/modules\/all\/, response: \nTraceback (most recent call last):\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/middleware\/errors.py\", line 164, in __call__\n    await self.app(scope, receive, _send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/middleware\/exceptions.py\", line 62, in __call__\n    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 64, in wrapped_app\n    raise exc\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 53, in wrapped_app\n    await app(scope, receive, sender)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 762, in __call__\n    await self.middleware_stack(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 782, in app\n    await route.handle(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 297, in handle\n    await self.app(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 77, in app\n    await wrap_app_handling_exceptions(app, request)(scope, receive, send)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 64, in wrapped_app\n    raise exc\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/_exception_handler.py\", line 53, in wrapped_app\n    await app(scope, receive, sender)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/starlette\/routing.py\", line 72, in app\n    response = await func(request)\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/fastapi\/routing.py\", line 315, in app\n    content = await serialize_response(\n  File \"\/layers\/paketo-buildpacks_pip-install\/packages\/lib\/python3.10\/site-packages\/fastapi\/routing.py\", line 155, in serialize_response\n    raise ResponseValidationError(\nfastapi.exceptions.ResponseValidationError: 1 validation errors:\n  {'type': 'model_attributes_type', 'loc': ('response',), 'msg': 'Input should be a valid dictionary or object to extract fields from', 'input': None, 'url': 'https:\/\/errors.pydantic.dev\/2.5\/v\/model_attributes_type'}\n\n\nsrc\/test\/java\/modules\/get\/modules-get.feature:13","executorName":"main","name":"Get a modules","startTime":1709609647847,"refId":"[1:10]","endTime":1709609652203,"exampleIndex":-1}],"callDepth":0,"name":"Get modules on request","description":"","resultDate":"2024-03-04 10:34:12 p.\u202Fm.","durationMillis":4340.5174,"passedCount":0}